src/rust/devcontainer-feature.json

```diff
@@ -39,7 +39,6 @@
         "vscode": {
             "extensions": [
                 "vadimcn.vscode-lldb",
-                "mutantdino.resourcemonitor",
```

Comment:
From https://marketplace.visualstudio.com/items?itemName=mutantdino.resourcemonitor, doesn't seem like this extension is useful or even essential for rust scenarios. Removing makes sense to me!

Thanks @nyuyuyu for bringing this up! ❇️ 

// Would appreciate any other thoughts from @bamurtaugh @joshspicer (Let me know if I am missing any use case)

Reply:
I agree with removing it - the extension seems to be a nice to have, but I don't see it as being essential for Rust development. It's also not mentioned in the VS Code docs on Rust: https://code.visualstudio.com/docs/languages/rust.